rockcraft.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: sdcore-upf-bess
-base: ubuntu@22.04
-version: '1.4.0'
+base: ubuntu@24.04
+version: '1.0.1'
```

Comment:
What's this version here? We probably want to use the same version here as the tag in the `upf` upstream project. In this case, that would be `1.5.0`

Reply:
This is the image for bessd and I believe it should follow the upstream version  which 1.0.1.  1.5.0 is the UPF version not bess version.

Reply:
ok, fair enough